qtazu/widgets/login.py

```diff
@@ -15,14 +91,21 @@ class Login(QtWidgets.QDialog):
     def __init__(self, parent=None, initialize_host=True):
         super(Login, self).__init__(parent)
 
-        self.setWindowTitle("Log-in to CGWire")
+        self.setWindowTitle("Connect your app to Kitsu")
```

Comment:
This seems rather specific for "your app". How about shortening it to: `Connect to Kitsu` or `Log-in to CGWire`? Or is there good reason to stick with this?

Reply:
To be honest I don't really know which name would be the best... @frankrousseau ?

Reply:
Because, saying that you log in to Kitsu is misleading. You will not have access to the same set of features. I'm afraid that it will be confusing for users.

Reply:
@frankrousseau So to understand you correctly you're more inclined to *Connect to*? Would *Connect to Kitsu* suffice? I'm find with all though.

Reply:
Yes, that sounds good!

Reply:
Nice :) I'll commit the changes tomorrow then !